app/controllers/posts_controller.rb

```diff
@@ -9,6 +9,12 @@ def index
 
   def show
     @comment_pagy, @comments = pagy(@post.comments.where(parent_id: nil), items: 5)
+
+    return unless user_signed_in? && current_user == @post.user
```

Comment:
are you sure about this condition?

Reply:
maybe just return unless current_user == @post.user

Reply:
move comments update inside condition, it would be more readable

Reply:
and make condition 
if current_user == @post.user
end